lib/pbench/test/unit/server/test_cache_manager.py

```diff
@@ -440,95 +487,258 @@ def mock_run(args, **kwargs):
+                            'children':
+                            {
+                                'f1.txt': {'details': <pbench.server.cache_manager.FileInfo object at 0x7fc4f1dc51d0>},
+                                'subdir13':
+                                {
+                                    'details': <pbench.server.cache_manager.FileInfo object at 0x7fc4f1dc4990>,
+                                    'children':
+                                    {
+                                        'subdir111':
+                                        {
+                                            'details': <pbench.server.cache_manager.FileInfo object at 0x7fc4f1dc4310>,
+                                            'children':
+                                            {
+                                                'f1_sym': { 'details': <pbench.server.cache_manager.FileInfo object at 0x7fc4f1db2250>}
+                                                'f1.txt': { 'details': <pbench.server.cache_manager.FileInfo object at 0x7fc4f1db01d0>}
+                                            }
+                                        }
+                                    }
+                                },
+                                'subdir12':
+                                {
+                                    'details': <pbench.server.cache_manager.FileInfo object at 0x7fc4f1dc48d0>
+                                },
+                                'subdir11':
+                                {
+                                    'details': <pbench.server.cache_manager.FileInfo object at 0x7fc4f1d98850>
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+            """
+
             m.setattr(Tarball, "__init__", TestCacheManager.MockTarball.__init__)
             m.setattr(Controller, "__init__", TestCacheManager.MockController.__init__)
-            m.setattr(Path, "glob", mock_glob)
-            m.setattr(os.path, "getmtime", lambda self: 1681814586.1138802)
-            m.setattr(Path, "is_dir", lambda self: True)
-            m.setattr(Path, "is_file", lambda self: True)
             tb = Tarball(tar, Controller(Path("/mock/archive"), cache, None))
-            tb.unpack()
-            assert tb.cachemap
-            assert tb.cachemap["A"]["dir1"]["details"].location == Path("/mock/A/dir1")
-            assert (
-                tb.cachemap["A"]["dir1"]["dir2"]["details"].timestamp
-                == "2023-04-18 16:13:06"
-            )
-            assert call == ["tar", "find"]
-            assert tb.unpacked == cache / "ABC" / tb.name
+            tar_dir = TestCacheManager.MockTarball.create_tarball("tar_name")
+            tb.cache_map(tar_dir)
+            assert tb.cachemap["tar_name"]["details"].type == "directory"
```

Comment:
This seems redundant with the identical check in `test_cache_map_traverse_cmap()`.